buddyboss/buddyboss-platform
# Fix - Redundant container when Creating a new Album

### All Submissions:


Fixes #2462 

### How to test the changes in this Pull Request:

1. Go to 'Photos'
2. Click on 'Create Album'
3. There should be no uploader popup visible in the background

### Proof Screenshots or Video
https://ibb.co/R6gvQPK

### Other information:
